eng/pipelines/prepare-release.yml

```diff
@@ -1,19 +1,17 @@
 stages:
 - stage: PrepareReleaseStage
   displayName: Release Preparation
-  ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')) }}:
-    dependsOn:
-    - publish_using_darc
   jobs:
   - job: PrepareReleaseJob
     displayName: Prepare release with Darc
-    pool:
-      ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        name: NetCore-Public
-        demands: ImageOverride -equals windows.vs2022.amd64.open
-      ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-        name: NetCore1ESPool-Internal
-        demands: ImageOverride -equals windows.vs2022.amd64
+    ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'],'refs/heads/juhoyosa/1es-migration'))) }}:
```

Comment:
Are we running this on the `juhoyosa/1es-migration` branch temporarily until the migration is complete?

Reply:
I removed the references to juan's branch. I'm running on my branch and getting some warnings which are fine.